cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/dto/ErrorResponseDTO.java

```diff
@@ -92,4 +92,9 @@ public void addErrorHolder( ErrorHolder errorHolder )
     {
         errorHolders.add( errorHolder );
     }
+
+    public ErrorHolder firstErrorHolder()
+    {
+        return this.errorHolders.get( 0 );
```

Comment:
`this.errorHolders` can be null

Reply:
nope, it's initialized and will throw IndexOutOfBoundsException if empty, fair enough?